### DISCUSSION TITLE
quic_stream_capacity 这个接口返回值的意义

### DISCUSSION THREAD

**Discussion by stopandjoin** [Tencent/tquic#228] [Q&A (问题和回答)] [answered]
/**
 * Return the stream’s send capacity in bytes.
 */
ssize_t quic_stream_capacity(struct quic_conn_t *conn, uint64_t stream_id);

以上这个接口，返回值capacity 是指的当前流的最大可发送码率， 还是当前总共发送了多少字节？

**Comment by glime-ai** [comment, marked as answer]
返回值 `capacity` 指的是当前流的可发送容量，即在当前时刻，这个流还能发送多少字节的数据。这不是指流的最大可发送码率，也不是指流总共已发送了多少字节，而是指在调用此函数的时刻，基于流控制和拥塞控制的限制，这个流还能发送多少额外的数据。

**Reply by iyangsj** [reply]
指当前流还可以写入的数据量
